docs/octopus-cloud/index.md

```diff
@@ -19,29 +19,29 @@ You can sign up for Octopus Cloud at [octopus.com/register](https://octopus.com/
 ## Create an Octopus Account
 
 An Octopus account lets you manage your instances of Octopus Cloud.
+:::hint
+Skip the steps below by registering using your Google or Microsoft account.
+:::
 
 1. Enter your name.
-1. Provide your email address and click **Create a password**. Please note, these credentials are for you Octopus Account. You will also [create credentials](#create-a-cloud-instance) for your Octopus Cloud instance, when you create it.
-1. On the next screen, provide your company name.
-1. Chose a secure password and enter it twice.
-1. Click **Create my Octopus account**.
+1. Provide your email address and create a password. Please note, these credentials are for your Octopus account and you will use them to log into your Octopus instance through single sign-on.
+1. On the next screen, verify your email address.
+1. After your email has been verified, you will be logged into your Octopus account.
 
 ## Create a Cloud Instance
 
-1. From the instances screen, click **Create cloud instance**.
+1. From the **products** screen, click **Start a free 30-day Cloud trial**.
 1. Enter an **instance name** for your Octopus Cloud instance.
 1. Choose a URL for the instance.
 1. Select the Cloud region for your instance. Currently the only option is **US - Oregon**.
 1. Click **Enter account details**.
 1. Create your first user for Octopus Cloud.
```

Comment:
>Create your first user for Octopus Cloud.

I think this no longer applies for v2 instances?

Reply:
great, I've updated this.